docs/hugo/content/introduction/breaking-changes/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Breaking Changes"
+linkTitle: "Breaking Changes"
+---
+We go to great lengths to avoid breaking changes as much as possible, as we're well aware that they can cause issues for our users. However, they do occasionally occur, so we've committed to providing good documentation each time this occurs.
+
+## Released Breaking Changes
+
+### v2.0.0
+
+Breaking changes are:
+
+* Upgrades from releases prior to v2.0.0-beta.5 are disallowed
+* Alpha CRD versions have been removed
+
+For more information see [v2.0.0 Breaking Changes](./breaking-changes-v2.0.0).
+
+### v2.0.0-beta.4
+
+In the `beta.4` release of Azure Service Operator (ASO) we pivoted to using Azure Swagger API Specifications as the sole source of truth for our code generator. This change brought with it a significant improvement in fidelity - the code we generate is now much closer to what the Azure Swagger API Specifications describe. Unfortunately, this change also brings with it a number of breaking changes requiring simple manual modifications to pre-existing resources.
+
+Breaking changes are:
+
+* The discriminator value for polymorphic types has been changed to match the name of the property used to specify that option.
+* Enumerated properties that previously had a base type but no enumeration values have been updated to include the enumeration values.
+* Objects that were incorrectly generated as nested properties have been directly inlined
+* ARM Id fields are now correctly identified as references, allowing linking to a resource in Kubernetes instead of only in Azure.
+* Status properties that cannot be set by the end user on a Spec that were included in the Spec in error
+* Sub-resources that were incorrectly inlined into the parent resource have been moved to a separate resource.
+* Properties that previously included on Spec but actually had no function have been removed.
+* Validation rules have been tightened, or added to properties that previously had no validation rules.
+* 
+For detailed information, including an exhaustive list of all affected resource properties, see [v2.0.0 Breaking Changes](./breaking-changes-v2.0.0-beta.4).
+
+## Upcoming Breaking Changes
+
+Beta CRD versions (any version with `v1beta` prefix) will be deprecated no sooner than **v2.3.0**. 
+We recommend you start using `v1api` prefixed versions now. 
+
+You can easily swap from a `v1beta` version to a `v1api` version by just replacing `v1beta` with `v1api` in your CRD YAML, the resource shapes are the same.
+
+## Policies
+
+Prior to the v2.0.0 GA release, our policy was *avoid breaking changes*. 
+We're now upgrading that to *no avoidable breaking changes*.
+
+If we do encounter potential breaking changes, we have a number of potential mitigations.
+
+### Conversion augmentation
```

Comment:
This feels very developer-facing rather than user-facing, as users may not understand how we do intra-version conversions (or that we even do).

I wonder if it would be better to outline the commitments we're giving first, before talkin about mitigations. I think those commitments are:
1. We will not introduce a breaking change in an existing version of a resource such as `v1api20230101`
2. We _may_ introduce breaking changes between versions of resources, such as between `v1api20230101` and `v1api20230201`. These two versions correspond to two different Azure API versions, and if the upstream Azure service introduces a breaking change in the newer of these two API versions we may pass that breaking change along. When this happens we will do our best to hide these breaking changes from our users (see mitigations below).

This more explicitly spells out what exactly "no avoidable breaking changes" means.

Reply:
Good idea. Adopted and expanded.